src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java

```diff
@@ -428,7 +405,10 @@ private boolean isMissingJavadocAllowed(final DetailAST ast) {
      */
     private boolean isContentsAllowMissingJavadoc(DetailAST ast) {
         return (ast.getType() == TokenTypes.METHOD_DEF || ast.getType() == TokenTypes.CTOR_DEF)
-                && (getMethodsNumberOfLine(ast) <= minLineCount || hasAllowedAnnotations(ast));
+                && (
+                        getMethodsNumberOfLine(ast) <= minLineCount
+                        || AnnotationUtil.hasAllowedAnnotations(ast, allowedAnnotations)
+                );
```

Comment:
This formatting looks weird imo.
@romani Whats your opinion?

Reply:
interesting concept to read it more clearly ...

@ngeor , can we make it like ?
```
 && (getMethodsNumberOfLine(ast) <= minLineCount
              || AnnotationUtil.hasAllowedAnnotations(ast, allowedAnnotations));
```

Reply:
of course!

Reply:
hm, now that I see it closer, this is an odd number of spaces. Putting the pipe under the 's' gives it 13 spaces indentation. 

Here it is with 4 spaces:

```
&& (getMethodsNumberOfLine(ast) <= minLineCount
    || AnnotationUtil.hasAllowedAnnotations(ast, allowedAnnotations));
```

8 spaces:

```
&& (getMethodsNumberOfLine(ast) <= minLineCount
        || AnnotationUtil.hasAllowedAnnotations(ast, allowedAnnotations));
```

and 12 spaces:

```
&& (getMethodsNumberOfLine(ast) <= minLineCount
            || AnnotationUtil.hasAllowedAnnotations(ast, allowedAnnotations));
```

take your pick :-)

Reply:
"Here it is with 4 spaces" is good.

Reply:
fixed